mobile/src/controllers/HealthPermissions.ts

```diff
@@ -0,0 +1,87 @@
+import { observable } from 'mobx';
+import { IUserNameProvider } from 'src/services/Notifications';
+import { ILocalSettingsController } from './LocalSettings';
+import { ThrottleAction } from 'common/utils/throttle';
+import { IDisposable } from 'common/utils/unsubscriber';
+import {auth, init, disconnectAndroid, getDOB, getAuthStatus} from 'src/helpers/health'
+import { Platform } from 'react-native';
+import logger, { createLogger } from 'common/logger';
+
+
+export class HealthPermissionsController implements IDisposable {
+
+    @observable
+    private _enabledByUser: boolean;
+
+    @observable
+    private _enabledByUserOG: boolean;
```

Comment:
just wondering what the 'OG' suffix means here. Originally enabled by user?

Reply:
Yes, maybe I should rename it to be more clearer